app/lib/features/space/pages/space_details_page.dart

```diff
@@ -264,6 +265,7 @@ class _SpaceDetailsPageState extends ConsumerState<SpaceDetailsPage> {
       TabEntry.overview => AboutSection(spaceId: widget.spaceId),
       TabEntry.news => NewsSection(spaceId: widget.spaceId),
       TabEntry.pins => PinListWidget(
+          pinListProvider: allPinListWithBookmarkFrontProvider(widget.spaceId),
```

Comment:
So that means space would only have access to bookmarked pins and not all of them ? Even if we want to see all of them??

Reply:
`allPinListWithBookmarkFrontProvider`

This is giving all pin list with bookmark pins are in front of link...